packages/histoire/src/node/vite.ts

```diff
@@ -105,9 +105,10 @@ async function mergeHistoireViteConfig (viteConfig: InlineConfig, ctx: Context)
 export async function getViteConfigWithPlugins (isServer: boolean, ctx: Context): Promise<InlineConfig> {
   const resolvedViteConfig = await resolveViteConfig(ctx)
 
-  const userViteConfig = await loadViteConfigFromFile({ command: ctx.mode === 'dev' ? 'serve' : 'build', mode: ctx.mode })
+  const userViteConfigFile = await loadViteConfigFromFile({ command: ctx.mode === 'dev' ? 'serve' : 'build', mode: ctx.mode })
+  const userViteConfig = mergeViteConfig(userViteConfigFile?.config ?? {}, { server: { port: 6006 } })
```

Comment:
Should be the opposite, `mergeViteConfig` signature is `(defaults, overrides)`

Reply:
The `userViteConfigFile` is the config of Vite (not Histoire), so I believe this is what should be overridden with port 6006.
Else, Histoire will use the Vite defined port (if none is specified for Histoire) instead of the default 6006.

Basically here's how it works:

config: none
result: Histoire port is 6006

config: Vite port is 1234
result: Histoire port is 6006

config: Histoire port is 5678
result: Histoire port is 5678

config: Vite port is 1234 and Histoire port is 5678
result: Histoire port is 5678

Reply:
Oh you're right, we merge histoire's vite config into it afterwards.